package.json

```diff
@@ -37,7 +37,7 @@
     "dist"
   ],
   "engines": {
-    "node": "16.14.2",
-    "npm": "8.5.0"
+    "node": ">=16.14.2 <20.12.2",
```

Comment:
Supporting a range here.

Reply:
That's cool. I didn't know we could do that.